<launch>
	<arg name="ctrl_topic" default="/vesc/high_level/ackermann_cmd_mux/input/nav0"/>
	<arg name="static_map" default="static_map"/>
	<!--<arg name="nn_model" default="weights.th"/>-->
	<arg name="nn_model" default="/home/nvidia/catkin_ws/src/lab3/src/model.th"/>


    <node pkg="lab4" type="MPPI.py" name="MPPI" output="screen">
		<param name="ctrl_topic" value="$(arg ctrl_topic)"/>
		<param name="static_map" value="$(arg static_map)"/>
		<param name="nn_model" value="$(arg nn_model)"/>
	</node>

</launch>
